jbpm-flow/src/test/java/org/jbpm/ruleflow/core/validation/RuleFlowProcessValidatorTest.java

```diff
@@ -125,4 +125,27 @@ public void testDynamicNodeValidationInDynamicProcess() throws Exception {
         assertEquals("Node 'MyDynamicNode' [1] Dynamic has no completion condition set", errors2[0].getMessage());
 
     }
+
+    @Test
+    public void testEmptyPackageName() throws Exception {
+        RuleFlowProcess process = new RuleFlowProcess();
+        process.setId("org.drools.core.process");
+        process.setName("Empty Package Name Process");
+        process.setPackageName("");
+        process.setDynamic(true);
+
+        ProcessValidationError[] errors = validator.validateProcess(process);
+        assertNotNull(errors);
+    }
+
+    @Test
+    public void testNoPackageName() throws Exception {
+        RuleFlowProcess process = new RuleFlowProcess();
+        process.setId("org.drools.core.process");
+        process.setName("No Package Name Process");
+        process.setDynamic(true);
+
+        ProcessValidationError[] errors = validator.validateProcess(process);
+        assertNotNull(errors);
+    }
```

Comment:
I am maybe overlooking something, but it seems that these tests are not really testing the fix? They would pass even when the fix would be removed? Should we perhaps assert specific content of the `errors` array?